### DISCUSSION TITLE
How do I prevent the session load in "PersistedLoadPre"

### DISCUSSION THREAD

**Discussion by Ajaymamtora** [olimorris/persisted.nvim#136] [Q&A] [answered]
Id like to check if any terminal buffers are open in this event, prompt the user, then if they reject, cancel the session load

```lua
  vim.api.nvim_create_autocmd({ "User" }, {
    pattern = "PersistedLoadPre",
    group = group,
    callback = function(session)
-- prompt -> exit
    end,
  })
```

**Comment by olimorris** [comment, marked as answer]
You can't. I'd advise creating your own function which checks for terminal buffers and then loads a session.
